xray/audit/python/python.go

```diff
@@ -93,8 +92,12 @@ func getDependencies(pythonTool pythonutils.PythonTool, requirementsFile string)
 	}
 	dependenciesGraph, directDependencies, err = pythonutils.GetPythonDependencies(pythonTool, tempDirPath, localDependenciesPath)
 	if err != nil {
-		audit.LogExecutableVersion("python")
-		audit.LogExecutableVersion(string(pythonTool))
+		if _, innerErr := audit.GetAndLogExecutableVersion("python"); innerErr != nil {
+			log.Error(innerErr)
```

Comment:
This will be available out of the box in go 1.20 (https://github.com/golang/go/issues/53435). I suggest we go through the code to update code like this after we upgrade